docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -267,385 +280,424 @@
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/Maps" target="_blank"><strong>Maps</strong></a></p>
-			<p>The Sample Maps application demonstrates how user can use the maps.</p>
+			<p>This sample application demonstrates how user can use the maps.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m16lescanner.jpg" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/Lescanner" target="_blank"><strong>Lescanner</strong></a></p>
-			<p>Lescanner application demonstrates how to scan the BLE devices.</p>
+			<p>This sample application demonstrates how to scan the BLE devices.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m17networkapp.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NetworkApp" target="_blank"><strong>NetworkApp</strong></a></p>
-			<p>NetworkApp demonstrates how user can use Connection, WiFi and WiFi Direct.<br />
-			User can get network information such as IP address, network state and available network lists and make Wi-Fi connection.</p>
+			<p>This sample application demonstrates how user can use Connection, Wi-Fi, and Wi-Fi Direct.<br />
+			User can get network information such as IP address, network state, and available network lists and make Wi-Fi connection.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m18downloader.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/Downloader" target="_blank"><strong>Downloader</strong></a></p>
-			<p>Downloader application demonstrates how user can download contents and get the download information.</p>
+			<p>This sample application demonstrates how user can download contents and get the download information.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m19pushreceiver.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/PushReceiver" target="_blank"><strong>PushReceiver</strong></a></p>
-			<p>PushReceiver application demonstrates how user can receive push notification from application server through push server.</p>
+			<p>This sample application demonstrates how user can receive push notification from application server through push server.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m20samplesync.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/SampleSync" target="_blank"><strong>SampleSync</strong></a></p>
-			<p>SampleSync application demonstrates how you can manage data synchronization schedule between target device and server.</p>
+			<p>This sample application demonstrates how you can manage data synchronization schedule between target device and server.</p>
```

Comment:
between the target device and the server.

Reply:
fixed